cmd/thanos/query.go

```diff
@@ -46,6 +53,15 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application, name string
 	clusterAdvertiseAddr := cmd.Flag("cluster.advertise-address", "explicit address to advertise in cluster").
 		String()
 
+	gossipInterval := cmd.Flag("cluster.gossip-interval", "interval between sending gossip messages. By lowering this value (more frequent) gossip messages are propagated across the cluster more quickly at the expense of increased bandwidth.").
+		Default(cluster.DefaultGossipInterval).Duration()
+
+	pushPullInterval := cmd.Flag("cluster.pushpull-interval", "interval for gossip state syncs . Setting this interval lower (more frequent) will increase convergence speeds across larger clusters at the expense of increased bandwidth usage.").
+		Default(cluster.DefaultPushPullInterval).Duration()
+
+	selectorLabels := cmd.Flag("selector-label", "query selector labels that will be exposed in info endpoint (repeated)").
+		PlaceHolder("<name>=\"<value>\"").Strings()
```

Comment:
Why "selector" and not "store"?

Reply:
Because we don't append this label to query result. See option C) on our slides, which makes external labels appended only by source nodes.